integration/client/signatures/rules_test.go

```diff
@@ -120,7 +120,7 @@ func TestImageAllowRules(t *testing.T) {
 	assert.Error(t, err, "should error since image is not covered by images scope of IAR")
 
 	// update image allow rule to cover that image
-	iar.Images = []string{tagName}
+	iar.Images = []string{"**"}
```

Comment:
Why this change?

Reply:
I will merge that change away with https://github.com/acorn-io/runtime/pull/2113 :+1: